src/table/__tests__/sortable-head-cell.test.js

```diff
@@ -20,6 +21,16 @@ describe('sortable-head-cell', () => {
     expect(down).toHaveLength(1);
   });
 
+  it('prop extendClick enable click on the complete cell', () => {
```

Comment:
Please duplicate 'calls provided onSort' and 'does not call if disabled' tests to ensure it works as-expected with this new flag